lib/popt.xml

```diff
@@ -12,11 +12,16 @@ strings into argv[] arrays using shell-like rules.
 
 This package contains the runtime library and locale data.</description>
   <package-implementation package="libpopt0"/>
+  <package-implementation distributions="Gentoo" package="dev-libs/popt"/>
+  <package-implementation package="popt"/>
   <implementation arch="Linux-i386" id="sha1new=362577ba8014c27fa70d458512af35b56cdf5ea6" released="2012-05-26" version="1.16-1">
       <archive href="libpopt0-i386-1.16-1.tar.bz2" size="51588"/>
-    </implementation>
+  </implementation>
   <implementation arch="Linux-x86_64" id="sha1new=856e07e63b6130b96ab39bf8dfc3ac5e3d18db96" released="2012-05-26" version="1.16-1">
       <archive href="libpopt0-amd64-1.16-1.tar.bz2" size="53008"/>
-    </implementation>
-
+  </implementation>
+  <implementation arch="Windows-i486" id="sha1new=d5bbadb4937a12bc359ec5a7b8b31386fddda0e3" langs="cs da de es eu_ES fi fr hu id is it ja ko no pl pt pt_BR ro ru sk sl sr sv tr uk zh zh_CN" license="MIT/X Consortium License" released="2003-10-04" version="1.8-1">
```

Comment:
I'd suggest removing the `langs` attribute as long as there are no other implementations with a different set of languages for 0install to potentially choose.

Reply:
done